packages/fluentui/react-northstar-prototypes/src/prototypes/AccessibleTeams/AccessibleTeams.tsx

```diff
@@ -0,0 +1,247 @@
+      event.target.dontResetNavBarItems = true;
+
+      // Determine and save the focused navBar items
+      const items = event.currentTarget.querySelectorAll('.item');
+      setNavBarItems(items);
+
+      // Find the navBar item with tabindex="0" and set the focused navBar item index accordingly
+      Array.from(items).forEach((item: HTMLElement, index) => {
+        // Begin forEach 1
+        const tabindex = item.getAttribute('tabindex');
+        if (tabindex === '0') {
+          // Begin if 1
+          setFocusedItemIndex(index);
+        } // End if 1
+      }); // End forEach 1
+    },
+    [navBarItems],
+  ); // End handleFocus
+
+  const handleBlur = React.useCallback(
+    event => {
+      if (!Array.from(navBarItems).includes(event.relatedTarget)) {
+        // Begin if 1
+        clearTimeout(timeout);
+      } // End if 1
+    },
+    [navBarItems],
+  ); // End handleBlur
+
+  return (
+    <>
+      <h1>Accessible Teams prototype</h1>
+      <EventListener type="keydown" listener={handleKeyDown} target={document} />
+
+      <button>Back</button>
+
+      <input
+        type="text"
+        role="combobox"
+        aria-controls=""
+        aria-expanded="false"
+        aria-label="Look for messages, files, and more. Or type / for a list of commands."
+      />
+
+      <button aria-haspopup="true" aria-describedby="profile-desc">
+        Profile, app settings, and more.
+      </button>
+      <div id="profile-desc" aria-hidden="true">
+        Your profile picture with status displayed as Available.
+      </div>
+
+      <h2>Navigation bar</h2>
+      <div
+        role="group"
+        aria-label="Navigation bar"
+        onFocus={handleFocus}
+        onBlur={handleBlur}
+        data-instruction="To navigate use the arrow keys"
+      >
+        <button className="item" tabIndex={0} aria-pressed="false">
+          Activities
+        </button>
+        <div>
+          <button className="item" tabIndex={-1} aria-pressed="true" aria-describedby="chats-desc">
+            Chats
+          </button>
+          <div id="chats-desc" style={srOnlyCss}>
+            7 conversations with new messages
+          </div>
+        </div>
+        <div>
+          <button className="item" tabIndex={-1} aria-pressed="false" aria-describedby="teams-desc">
+            Teams
+          </button>
+          <div id="teams-desc" style={srOnlyCss}>
+            4 teams with new messages
+          </div>
+        </div>
+        <button className="item" tabIndex={-1} aria-pressed="false">
+          Calendar
+        </button>
+      </div>
+
+      <button aria-haspopup="menu">Chat</button>
+
+      <button>Filter</button>
+
+      <button>New chat</button>
+
+      <h2>Chats</h2>
+      <div role="tree" aria-label="Chat list">
+        <div role="treeitem">Tyler Skywalker</div>
+        <div role="treeitem" tabIndex={0}>
+          Michael Night
+        </div>
+      </div>
+
+      <div tabIndex={0}>Michael Night</div>
```

Comment:
please, I am not sure what is the purpose of the element?, as it is missing role and it is focusable <div>

Reply:
Thanks, this should be a button with aria-haspopup="true"